polygeist/tools/cgeist/Options.h

```diff
@@ -127,6 +127,11 @@ static llvm::cl::opt<bool> OpenMPOpt("openmp-opt", llvm::cl::init(true),
 static llvm::cl::opt<bool> EnableLICM("licm", llvm::cl::init(true),
                                       llvm::cl::desc("Turn on LICM"));
 
+static llvm::cl::opt<bool>
+    EnableLoopInternalization("enable-loop-internalization",
```

Comment:
Absolute nit/name shedding, but should we call the flag `loop-internalization` (i.e., remove `enable-`) to better match other optimization flags?